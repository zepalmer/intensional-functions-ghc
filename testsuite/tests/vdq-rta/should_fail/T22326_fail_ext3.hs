{-# LANGUAGE ExplicitNamespaces #-}
{-# LANGUAGE RequiredTypeArguments #-}
{-# LANGUAGE NoScopedTypeVariables #-}

module T22326_fail_ext3 where

-- Binding a type variable requires ScopedTypeVariables.
f :: forall x -> Show x => x -> String
f (type t) = show @t