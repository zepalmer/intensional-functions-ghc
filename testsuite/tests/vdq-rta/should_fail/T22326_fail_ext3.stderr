
T22326_fail_ext3.hs:9:4: error: [GHC-13911]
    Illegal type variable binding in a pattern: ‘t’
    Suggested fix: Perhaps you intended to use ScopedTypeVariables
